pkg/config/config_test.go

```diff
@@ -214,11 +216,19 @@ func TestComposeConfig(t *testing.T) {
 				},
 			},
 		},
+		featureGates: featuregates.NewHardcodedFeatureGateAccess(
+			[]configv1.FeatureGateName{"CloudDualStackNodeIPs", "ChocobombVanilla", "ChocobombStrawberry"},
+			[]configv1.FeatureGateName{"ChocobombBlueberry", "ChocobombBanana"},
+		),
 		expectConfig: OperatorConfig{
 			ManagedNamespace: defaultManagementNamespace,
 			ImagesReference:  defaultImagesReference,
 			PlatformStatus:   &configv1.PlatformStatus{Type: configv1.OpenStackPlatformType},
 			IsSingleReplica:  true,
+			// We only see CloudDualStackNodeIPs returned here because openshift/api/config defines
+			// white-listed features that are allowed to be used by cloud providers. Anything that
+			// is not defined there won't be passed to the cloud provider.
```

Comment:
Is this comment in need of update?

Reply:
Yes, once we settle on the implementation I will fix all docstrings